### DISCUSSION TITLE
Storage used in drive

### DISCUSSION THREAD

**Discussion by mawoka-myblock** [deta/cloud-docs#377] [Help]
Where can I see how much storage I've used/how much I still have left?

**Comment by lemonyte** [comment]
Quoting mustafa from [this](https://github.com/orgs/deta/discussions/325#discussioncomment-3187408) answer,

> drive has a limit of 10gb per account. there's no way to check how much you have used at the moment.

**Comment by AyushSehrawat** [comment]
@mawoka-myblock like @LemonPi314 said, there are no inbuilt methods, though manual methods like whenever putting a file in drive, save it's storage in Base, and when deleting remove it.. To check just fetch all and sum..

**Comment by jnsougata** [comment]
also you can just make a json file containing the file metadata... for example 
```
{
    "id": <file_id>,
    "size": <file_size>,
    "upload_date": <date>
}
```
and you can just store it as `metadata/<file_name>.json` this will give you ease to access the metadata. for each file upload you can do this. you can use this metadata json in future for any type of info you want or as you are suggested above use a base to store metadata

**Comment by mawoka-myblock** [comment]
That's ridiculous, that you can't see that. Imagine an email-provider that only provides 500mb of storage. You can't see how much storage you've used and if you are over 500mb, you can't receive any emails. That's not OK.

**Reply by AyushSehrawat** [reply]
It may be added in future but right now there's no such option.... You can try the manual way, it's just few more lines

**Reply by lemonyte** [reply]
Deta cloud is in beta after all... The team's got a platform update coming sometime so maybe it will be implemented then.
